### PR TITLE
Style

### PR DESCRIPTION
**Pull request by mnatour2**

### CODE REVIEW

File: Login/index.html [hunk 1]
```diff
@@ -3,29 +3,61 @@
...
 <body>
-    <header id="header1">
+    <!-- <header id="header1">
         <nav id="topnav">
             <a class="topnav-item" href="#">Sign Up</a>
             <a class="active topnav-item" href="#">Login</a>
             <a class="topnav-item" href="#">Home</a>
         </nav>
-    </header>
+    </header> -->
```
Comment (MajdTaweel) [inline Login/index.html:21]: Remove commented lines.

File: Login/styleSheet.css [hunk 2]
```diff
@@ -5,78 +5,184 @@ html {
...
-}
+} */
 
 /* Add a color to the active/current link */
-#topnav a.active {
+/* #topnav a.active {
   background-color: #04aa6d;
   color: white;
-}
+}  */
```
Comment (MajdTaweel) [inline Login/styleSheet.css:42]: Remove commented lines.

**Review by MajdTaweel — CHANGES_REQUESTED** [review]
Buttons' cursors should be pointers, instead, they're the default cursor.

File: Login/index.html (hunk 1 above)
Comment (mnatour2) [reply Login/index.html:21]: I'm gonna reuse them later on for a top nav.

File: Login/styleSheet.css (hunk 2 above)
Comment (mnatour2) [reply Login/styleSheet.css:42]: I'm gonna reuse them later on for a top nav.

**Comment by mnatour2** [comment]
Don't accept the merge request because I'm gonna do the register on the same branch

**Review by MajdTaweel — APPROVED** [review]

**Pull request merged**